### PR TITLE
[Update] Update the version requirement of mmdet in docker

### PR DESCRIPTION
**Pull request by Mountchicken**
#965

### CODE REVIEW

**Comment by xinke-wang** [comment]
I think we may also need to update the dependencies [here](https://mmocr.readthedocs.io/en/latest/install.html). The main branch requires a mmdet version >= 2.21. Also, there is a typo here, the `master` should be revised to `main` for our repo.

**Review by xinke-wang — APPROVED** [review]

File: docs/en/install.md [hunk 1]
```diff
@@ -16,8 +16,8 @@ MMOCR has different version requirements on MMCV and MMDetection at each release
 
 | MMOCR        | MMCV                   | MMDetection               |
 | ------------ | ---------------------- | ------------------------- |
-| master       | 1.3.8 <= mmcv <= 1.6.0 | 2.14.0 <= mmdet <= 3.0.0  |
-| 0.5.0        | 1.3.8 <= mmcv <= 1.5.0 | 2.14.0 <= mmdet <= 3.0.0  |
+|   main       | 1.3.8 <= mmcv <= 1.6.0 | 2.21.0 <= mmdet <= 3.0.0  |
+| 0.5.0        | 1.3.8 <= mmcv <= 1.5.0 | 2.21.0 <= mmdet <= 3.0.0  |
```
Comment (gaotongxiao) [inline docs/en/install.md:20]: #944 was merged after the release of 0.5.0. 0.5.0 is still compatible with mmdet>=2.14.0

**Review by gaotongxiao — CHANGES_REQUESTED** [review]

File: docs/en/install.md (hunk 1 above)
Comment (gaotongxiao) [reply docs/en/install.md:20]: And there are extra spaces before "main"

**Review by gaotongxiao — COMMENTED** [review]
Forgot to mention a place needed to be changed as well: https://github.com/open-mmlab/mmocr/blob/main/mmocr/__init__.py#L59

**Comment by Mountchicken** [comment]
> Forgot to mention a place needed to be changed as well: https://github.com/open-mmlab/mmocr/blob/main/mmocr/__init__.py#L59

Done

**Review by gaotongxiao — APPROVED** [review]

**Pull request merged**